core/domain/stats_services.py

```diff
@@ -313,13 +313,25 @@ def update_exp_issues_for_new_exp_version(
                 old_to_new_state_names[state_name])
 
         for action in playthrough.actions:
-            state_name = (
-                action.action_customization_args['state_name']['value'])
-            action.action_customization_args['state_name']['value'] = (
-                state_name if state_name not in old_to_new_state_names else
-                old_to_new_state_names[state_name])
-
-        playthrough_model.populate(**playthrough.to_dict())
+            action_customization_args = action.action_customization_args
+
+            if 'state_name' in action_customization_args:
+                state_name = action_customization_args['state_name']['value']
+                action_customization_args['state_name']['value'] = (
+                    state_name if state_name not in old_to_new_state_names else
+                    old_to_new_state_names[state_name])
+
+            if 'dest_state_name' in action_customization_args:
+                dest_state_name = (
+                    action_customization_args['dest_state_name']['value'])
+                action_customization_args['dest_state_name']['value'] = (
+                    dest_state_name
+                    if dest_state_name not in old_to_new_state_names else
+                    old_to_new_state_names[dest_state_name])
+
+        playthrough_model.issue_customization_args = (
+            playthrough.issue_customization_args)
+        playthrough_model.actions = [a.to_dict() for a in playthrough.actions]
```

Comment:
Could we name ``a`` as ``action``, instead, for clarity? Not strongly fixed on this, but it's nice per consistency with ``issue`` for-loop above.

Reply:
Done.